### ISSUE TITLE
3.2.0 Beeper Configuration bugs

### ISSUE THREAD

**Issue by swilkens** [betaflight/betaflight-configurator#596]
Configurator: 3.2.0
Firmware: 3.2.0-RC2

After looking here: https://github.com/betaflight/betaflight/issues/3754 

I dug a little deeper and found some funkyness going on with the beeper configuration. It seems that the GUI switches do not properly control the correct features in the firmware. This can be seen by comparing the GUI config to the CLI dump. See below for examples.

With all beep switches on:
![1beep](https://user-images.githubusercontent.com/13329383/28982686-0ae4ac82-7957-11e7-8ec7-100997842be0.PNG)

We see this:
```
beeper GYRO_CALIBRATED
beeper RX_LOST
beeper RX_LOST_LANDING
beeper DISARMING
beeper ARMING
beeper ARMING_GPS_FIX
beeper BAT_CRIT_LOW
beeper BAT_LOW
beeper GPS_STATUS
beeper RX_SET
beeper ACC_CALIBRATION
beeper ACC_CALIBRATION_FAIL
beeper READY_BEEP
beeper MULTI_BEEPS
beeper DISARM_REPEAT
beeper ARMED
beeper SYSTEM_INIT
beeper ON_USB
beeper BLACKBOX_ERASE
```

However, with all beep switches off:
![2beep](https://user-images.githubusercontent.com/13329383/28982758-3ae69ac6-7957-11e7-9528-a3b8875ef68e.PNG)

It seems that  GPS_STATUS and DISARM_REPEAT aren't disabled.
```
beeper -GYRO_CALIBRATED
beeper -RX_LOST
beeper -RX_LOST_LANDING
beeper -DISARMING
beeper -ARMING
beeper -ARMING_GPS_FIX
beeper -BAT_CRIT_LOW
beeper -BAT_LOW
beeper GPS_STATUS ====
beeper -RX_SET
beeper -ACC_CALIBRATION
beeper -ACC_CALIBRATION_FAIL
beeper -READY_BEEP
beeper -MULTI_BEEPS
beeper DISARM_REPEAT ====
beeper -ARMED
beeper -SYSTEM_INIT
beeper -ON_USB
beeper -BLACKBOX_ERASE
```

Even stranger, if I then toggle GPS_STATUS and DISARM_REPEAT in the GUI:
![3beep](https://user-images.githubusercontent.com/13329383/28982966-daef646c-7957-11e7-9c5d-da42af63532c.PNG)

It seems that the GPS_STATUS switch and the DISARM_REPEAT switch in the GUI actually control completely different features:
```
beeper -GYRO_CALIBRATED
beeper -RX_LOST
beeper -RX_LOST_LANDING
beeper -DISARMING
beeper -ARMING
beeper ARMING_GPS_FIX ====
beeper -BAT_CRIT_LOW
beeper -BAT_LOW
beeper GPS_STATUS ====
beeper -RX_SET
beeper ACC_CALIBRATION ====
beeper -ACC_CALIBRATION_FAIL
beeper -READY_BEEP
beeper -MULTI_BEEPS
beeper DISARM_REPEAT ====
beeper -ARMED
beeper -SYSTEM_INIT
beeper -ON_USB
beeper -BLACKBOX_ERASE
```

**Comment by swilkens** [comment]
And just to confirm, this still happens in the current master (I cloned the repo on commit 92214e5ad0d705acacb5728f68644a661de065eb) of the configurator.

**Comment by swilkens** [comment]
This appears to have been fixed on configurator 3.2.1 in combination with FW 3.2.0-RC3

**Comment by mikeller** [comment]
@swilkens: Can you please close the ticket then?
